full/using-dev-containers-vscode/translations/es/README.md

```diff
@@ -0,0 +1,78 @@
+
+Después, aprenderás a como [utilizar la extensión de Contenedores Remotos en Visual Studio Code](https://docs.microsoft.com/gl-es/learn/modules/use-docker-container-dev-env-vs-code/3-use-as-development-environment?WT.mc_id=academic-55190-ornella). Serás capaz de entender como Docker y Visual Studio Code hacen posible tener proyectos contenidos en sus propios entornos preconfigurados y en contenedores mediante esta extensión.
+
+## Añade un contenedor de desarrollo a un proyecto existente
+
+Ahora que ya tienes lista la extensión, [añadirás un contenedor de desarrollo y abriras el proyecto de ejemplo](https://docs.microsoft.com/gl-es/learn/modules/use-docker-container-dev-env-vs-code/4-exercise-add-development-container?WT.mc_id=academic-55190-ornella) en el contenedor.
+
+## Personaliza las configuraciones del proyecto y del editor de código
+
+Después de configurar el contenedor de desarrollo para el proyecto, aprenderás a como [personalizar y automatizar toda la configuración del proyecto](https://docs.microsoft.com/gl-es/learn/modules/use-docker-container-dev-env-vs-code/5-customize-settings?WT.mc_id=academic-55190-ornella) utilizando el [archivo devcontainer.json](https://docs.microsoft.com/gl-es/learn/modules/use-docker-container-dev-env-vs-code/6-exercise-customize-settings?WT.mc_id=academic-55190-ornella).
+
+## Añade software a un contenedor existente
+
+Hasta ahora has aprendido a como añadir un contenedor de desarrollo preconfigurado y personalizarlo utilizando el archivo devcontainer.json. Ahora, aprenderás a como [añadir software](https://docs.microsoft.com/gl-es/learn/modules/use-docker-container-dev-env-vs-code/7-add-software?WT.mc_id=academic-55190-ornella) más allá del que está disponible en esos contenedores de desarrollo preconfigurados para mejorar aún más tu experiencia. Aplica lo aprendido [utilizando un archivo de Docker (Dockerfile) para añadir Node.js a tu contenedor](https://docs.microsoft.com/gl-es/learn/modules/use-docker-container-dev-env-vs-code/8-exercise-add-software?WT.mc_id=academic-55190-ornella).
+
+
+## Prueba
+
+[Prueba de conocimientos](https://docs.microsoft.com/gl-es/learn/modules/use-docker-container-dev-env-vs-code/9-knowledge-check?WT.mc_id=academic-55190-ornella)
+
+## Próximos pasos
+
+- [Tutorial de contenedores remotos](https://code.visualstudio.com/docs/remote/containers-tutorial?WT.mc_id=academic-55190-ornella)
+- [Aprende a crear un contenedor de desarrollo](https://code.visualstudio.com/docs/remote/create-dev-container?WT.mc_id=academic-55190-ornella)
+- [Remoto principal - documentación de contenedores](https://code.visualstudio.com/docs/remote/containers?WT.mc_id=academic-55190-ornella)
+- [Cómo pueden utilizar los estudiantes los contenedores de desarrollo](https://www.youtube.com/watch?v=Uvf2FVS1F8k)
+
+## Práctica
+
+Prueba tu conocimiento, intenta crear tu propio contenedor de desarrollo lleno de tus herramientas de desarrollo favoritas. ¡Compártelo con un amigo!
+
```

Comment:
## Agradecimientos 💖 
¡Muchísimas gracias a [Alfonso Amaya](https://www.linkedin.com/in/alfonso-camargo/) @Alfonso-AmayaC por habernos ayudado a traducir todo este contenido a español!